jobs.md

```diff
@@ -5,9 +5,9 @@ title: RSE Opportunities
 
 ## Current RSE openings
 
-1. [Research Software Engineer](https://career4.successfactors.com/sfcareer/jobreqcareer?jobId=158&company=utbattelleP&username=Oakridge), Oak Ridge National Laboratory
-2. [Research Software Engineer/Computer Scientist](https://lanl.jobs/los-alamos-nm/research-software-engineercomputer-scientist-2-scientist-2/48A92E8C0DF947858E22C19FCCAB9C5D/job/), Los Alamos National Laboratory
-3. [Research Software Engineer/Computer Scientist](https://lanl.jobs/los-alamos-nm/research-software-engineercomputer-scientist-2-scientist-2/0B2E362EF2FF4AF2A94C259A695AD09C/job/), Los Alamos National Laboratory
+1. [Research Software Engineer/Computer Scientist](https://lanl.jobs/los-alamos-nm/research-software-engineercomputer-scientist-2-scientist-2/48A92E8C0DF947858E22C19FCCAB9C5D/job/), Los Alamos National Laboratory
```

Comment:
Is [this link](https://lanl.jobs/los-alamos-nm/research-software-engineercomputer-scientist-2-scientist-2/48A92E8C0DF947858E22C19FCCAB9C5D/job/) correct? I got a 404.

Reply:
Wtf?  What happened.  I didn't touch it

Reply:
Did the link change? Maybe this one? https://lanl.jobs/los-alamos-nm/research-software-engineercomputer-scientist-2-scientist-2/0B2E362EF2FF4AF2A94C259A695AD09C/job/

I bet you they have some kind of unique id (the bit before /job) that coincides with a version, and it was changed. I don't see anything like a permalink. Either way, I don't think you did anything.

Reply:
That's odd, it doesn't work for me either, and I tried both the before and after version.  Something apparently changed on the LANL end.  I'll take an issue to check on it, but in the meantime we shouldn't hold up Ian's PR for this.